tests/Shared/WorkloadTesting/AspireProject.cs

```diff
@@ -0,0 +1,295 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit.Abstractions;
+
+namespace Aspire.Workload.Tests;
+
+public class AspireProject : IAsyncDisposable
+{
+    public static Lazy<HttpClient> Client => new(CreateHttpClient);
+    public static string GetNuGetConfigPathFor(string targetFramework) =>
+        Path.Combine(BuildEnvironment.TestDataPath, "nuget8.config");
+
+    public Process? Process { get; private set; }
```

Comment:
Is there really only one `Process` for an "AspireProject"? Should this be `AppHostProcess`?

Reply:
hm good point. Right now we are tracking the only process that we start, but we could be tracking the others too. I'll fix.